lib/event_store_http_client/events/write.rb

```diff
@@ -14,14 +14,12 @@ class Write
         dependency :settings
         dependency :client
 
-        def initialize(type, data, stream_name, version)
```

Comment:
The initializer would seem to be more desirable unless the instance is intended to be re-used with other type, data, stream name, and version values.